scipy/stats/_continuous_distns.py

```diff
@@ -8200,15 +8200,14 @@ class truncnorm_gen(rv_continuous):
     Notes
     -----
     The standard form of this distribution is a standard normal truncated to
-    the range [a, b] --- notice that a and b are defined over the domain of the
-    standard normal.  To convert clip values for a specific mean and standard
-    deviation, use::
-
-        a, b = (myclip_a - my_mean) / my_std, (myclip_b - my_mean) / my_std
-
-    `truncnorm` takes :math:`a` and :math:`b` as shape parameters.
-
-    %(after_notes)s
+    the range ``[a, b]``, where ``a`` and ``b`` are user-provided shape
```

Comment:
```suggestion
    the range ``[a, b]``, where ``a`` and ``b`` are user-provided shape
```
I know that technically `a` and `b` should have single-backticks, not double, because they are parameters. The stylization of single-backticked text in italics would be particularly confusing here IMO, so I broke the rule. There were bigger problems before: no backticks around inline "code" `[a, b]`, no backticks around parameters `a` and `b`, and references to undefined mathematical symbols *a* and *b*, so hopefully that part of this PR can be considered an improvement.